src/emu/bus/a2bus/a2arcadebd.c

```diff
@@ -2,7 +2,7 @@
 
     a2arcadeboard.c
 
-    Implementation of the Third Millenium Engineering Arcade Board
+    Implementation of the Third Millennium Engineering Arcade Board
```

Comment:
Third Millenium Engineering is a trademark and is spelled with a single 'n'.  Please check properly before attempting to 'correct' the spelling of names/trademarks.